app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
Do not use single-letter variable names in your code, be more explicit in naming. Besides that, you could implement this task in one line using chaining and built-in string methods.